### DISCUSSION TITLE
Can I edit the default template?

### DISCUSSION THREAD

**Discussion by pentest499** [pwndoc/pwndoc#103] [Q&A]
Where do I need to edit the code to do this?

Thanks in advance

**Comment by pentest499** [comment]
If I do any changes lets say just increased font size to 1 point, the report generator giving error as "undefined". Please suggest me where to do changes to fonts.

**Reply by yeln4ts** [reply]
These changes should be done in the docx template. Can you check the error log when generating ? 
Launch the command `docker-compose logs -f pwndoc-backend`
Try generating the report
Paste the errors here
